sechub-pds-solutions/gosec/tests/README.adoc

```diff
@@ -0,0 +1,10 @@
+= System Tests
+
+== Steps
+
+1. copy `sechub-pds-tools-cli-0.0.0.jar` into this folder.
```

Comment:
Where do I get this file?
Or how can I create it?

Reply:
Good question. We should release `sechub-pds-tools-cli` again.

Reply:
I just pretended in the example, that we released sechub-pds-tools-cli-1.0.0.jar`.